pkg/applicationserver/io/packages/loradms/v1/api/uplinks_test.go

```diff
@@ -109,6 +109,6 @@ func toFloat64(x float64) *float64 {
 	return &x
 }
 
-func toHex(x objects.Hex) *objects.Hex {
+func toHexPtr(x objects.Hex) *objects.Hex {
```

Comment:
nit; it would be `hexPtr()`

You don't need to change this, but it would be consistent with the codebase

Reply:
`$ grep -R -E 'func (?:[a-zA-Z0-9]+)Ptr' pkg`

Reply:
No, I'll change it for consistency. I've misread the suggestion in the other PR as 'add Ptr', that's on me.